crates/cli/src/package.rs

```diff
@@ -50,9 +50,10 @@ async fn internal_fetch(
     let store_folder_name = package_version.to_store_name();
 
     // TODO: skip when it already exists in store?
+    let store_dir = Arc::new(config.store_dir.clone().to_path_buf());
     let cas_paths = download_tarball_to_store(
         tarball_cache,
-        Arc::new(config.store_dir.to_path_buf()),
+        &store_dir,
```

Comment:
This is not what I meant. You just moved the cloning up. I wanted you to edit `Npmrc::store_dir`.